dpbench/benchmarks/pairwise_distance/pairwise_distance_numba_dpex_n.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2022 Intel Corp.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import numpy as np
+
+import numba as nb
+
+# target implementation
+# commented due to dpnp is not supporting yet
```

Comment:
I'd say we add the numpy implementation and its fine if we get a failure in dpbench. We can then ge tit fixed in dpnp.

Reply:
I feel we should keep working version for now until dpnp has the fix, otherwise the dpbench pipeline will be always broken. I can create an issue for updating this part once dpnp adds the support. Will that work?

Reply:
Ok. When you create an issue for dpnp, can you also do the same in dpbench and link it to the dpnp issue; lest we forget to change this implementation after dpnp adds support. Thanks.